peewee_async.py

```diff
@@ -830,26 +735,42 @@ def execute_sql(self, *args, **kwargs):
                         (str(args), str(kwargs)))
         return super().execute_sql(*args, **kwargs)
 
-    async def aio_execute(self, query):
+    async def as_async_query_wrapper(self, cursor, query):
```

Comment:
Maybe we shold make it classmethod of AsyncQueryWrapper? Something like await AsyncQueryWrapper.create(cursor, query)